packages/account/src/Components/proof-of-identity-container/proof-of-identity-container.scss

```diff
@@ -1,17 +0,0 @@
-.manual-poi {
```

Comment:
Why are we deleting this? This is being used in poi/status/unsupported/unsupported.jsx?

Reply:
all these css styles are duplicated from unsupported.scss.
proof-of-identity-container.scss isn’t used anywhere